### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JamesCullen2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,8 +2,106 @@
 <html lang="en">
 <head>
     <title>James Cullen Portfolio</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body>
-
+    <header>
+        <nav>
```
Comment (charliecog) [inline index.html]: doesnt seem like its being used - potentially remove

File: index.html [hunk 2]
```diff
@@ -2,8 +2,106 @@
...
+            <div>
```
Comment (charliecog) [inline index.html]: doesnt seem like its being used - potentially remove

File: index.html [hunk 3]
```diff
@@ -2,8 +2,106 @@
...
+                    Initially, in 2015, I founded a digital tech start-up, Unity-Hub.
+                    The aim of the company was to develop a Web App that makes business meetings more productive.
+                    I secured equity investment from Webstart Bristol to develop my idea and earned a place in
+                    the first Entrepreneurial Spark incubator in Bristol. After receiving the investment I
+                    outsourced design and development work in order to prove concept and raise more investment.
+                    Employing the services of a front-end developer - and working alongside them - gave me a
+                    fantastic insight into coding and beyond, such as user-profiling and agile methodology.
+                    As a non-technical director, During this time, I became fascinated with the software
+                    development side of the project; I love finding solutions to problems.
+                    Ultimately, I decided to learn how to build software to ‘scratch my own itch’.</p>
```
Comment (charliecog) [inline index.html]: move closing </p> tag onto its own line to make it easily scannable

File: index.html [hunk 4]
```diff
@@ -2,8 +2,106 @@
...
+                            <img src="nav-bar-icon-about-me-link.png" alt="James Cullen About Me">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main class="container">
+        <section id="about">
+            <article>
```
Comment (charliecog) [inline index.html]: could produce the same effect without the use of articles as they are not doing anything

File: index.html [hunk 5]
```diff
@@ -2,8 +2,106 @@
...
+                    I secured equity investment from Webstart Bristol to develop my idea and earned a place in
+                    the first Entrepreneurial Spark incubator in Bristol. After receiving the investment I
+                    outsourced design and development work in order to prove concept and raise more investment.
+                    Employing the services of a front-end developer - and working alongside them - gave me a
+                    fantastic insight into coding and beyond, such as user-profiling and agile methodology.
+                    As a non-technical director, During this time, I became fascinated with the software
+                    development side of the project; I love finding solutions to problems.
+                    Ultimately, I decided to learn how to build software to ‘scratch my own itch’.</p>
+            </article>
+            <article>
```
Comment (charliecog) [inline index.html]: see above about articles

File: index.html [hunk 6]
```diff
@@ -2,8 +2,106 @@
...
+                <h1>my skills</h1>
+                <p>I enrolled on the Mayden Academy software development programme in February 2018.
+                    This began a four month intensive course of advanced software development, including: HTML, CSS,
+                    CSM, SQL, JS, MVC & PHP, Node.js and Zend.</p>
+            </article>
+            <article>
```
Comment (charliecog) [inline index.html]: see above about articles

File: index.html [hunk 7]
```diff
@@ -2,8 +2,106 @@
...
+                <p>I am a very proud husband and father of a one year-old baby boy, Joshua! We live in Keynsham
+                    in the first house we have ever owned. My wife, Hannah, and I met in Mexico in 2003 while we were
+                    volunteering for a charity organisation. My wife is originally from Bristol area whereas I was
+                    born and raised on the Merseyside. I enjoy keeping myself fit and active; I currently train at
+                    CrossFit and play for Keynsham Hockey Club.
+                    We are blessed with many great friends and family whom we share time with regularly.</p>
+            </article>
+        </section>
+        <section>
+            <article>
```
Comment (charliecog) [inline index.html]: see above about articles

File: index.html [hunk 8]
```diff
@@ -2,8 +2,106 @@
...
+                <h1 id="portfolio">websites</h1>
+                <a class="image-icons" target="_blank" href="http://oratemate.com/">
+                    <img src="oratemate-website-image-1.png" alt="OrateMate Limited">
+                </a>
+                <div>
```
Comment (charliecog) [inline index.html]: dont need div - could just rely on p tag

File: index.html [hunk 9]
```diff
@@ -2,8 +2,106 @@
...
+                    <p>Here's a website that I built for my training / coaching company, OrateMate.</p>
+                </div>
+            </article>
+            <article>
+                <h1>web apps</h1>
+                <a class="image-icons" target="_blank" href="#">
+                    <img src="web-app-image-2.png" alt="Unity-Hub">
+                </a>
+                <div>
```
Comment (charliecog) [inline index.html]: dont need div - could just rely on p tag

File: index.html [hunk 10]
```diff
@@ -2,8 +2,106 @@
...
+            <article>
+                <h1 id="portfolio">websites</h1>
+                <a class="image-icons" target="_blank" href="http://oratemate.com/">
+                    <img src="oratemate-website-image-1.png" alt="OrateMate Limited">
+                </a>
+                <div>
+                    <p>Here's a website that I built for my training / coaching company, OrateMate.</p>
+                </div>
+            </article>
+            <article>
```
Comment (charliecog) [inline index.html]: see above about articles

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: Courier, sans-serif;
+}
+
+header {
+    background-image: url(mobile_code_image.jpg);
```
Comment (charliecog) [inline styles.css]: missing " " from around the url - best practice

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+    header {
+        background-image: url(tablet_code_image.jpg);
+    }
+    li {
+        list-style-type: none;
+        margin-right: 10px;
+        text-align: left;
+    }
+    .footer-links {
+        overflow: hidden;
```
Comment (charliecog) [inline styles.css]: remove this to fix the linkedin logo being cut off at bottom of screen when screen resized

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+        position: relative;
```
Comment (charliecog) [inline styles.css]: not being used - remove

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+        display: inline-block;
```
Comment (charliecog) [inline styles.css]: not being used - remove

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+    header {
+        background-image: url(large_code_skills.jpg);
+    }
+    li {
+        float: right;
+        list-style: none;
+        margin-right: 10px;
+    }
+    .footer-links {
+        overflow: hidden;
```
Comment (charliecog) [inline styles.css]: not being used - remove

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+        position: relative;
```
Comment (charliecog) [inline styles.css]: not being used - remove

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+        display: inline-block;
```
Comment (charliecog) [inline styles.css]: not being used - remove

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+        text-align: center;
+    }
+    .footer-nav-buttons {
+        width: 30%;
+    }
+ }
+
+
+
+
```
Comment (charliecog) [inline styles.css]: remove all of this whitespace

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,144 @@
...
+/*Tablet */
+
+@media only screen and (min-width: 600px) {
+    header {
+        background-image: url(tablet_code_image.jpg);
+    }
+    li {
+        list-style-type: none;
```
Comment (charliecog) [inline styles.css]: already declared above - not necessary

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,144 @@
...
+
+.footer-links {
+    overflow: hidden;
+    position: relative;
+    display: inline-block;
+    width: 80%;
+    min-height: 30vh;
+}
+
+
```
Comment (charliecog) [inline styles.css]: remove whitespace lines

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,144 @@
...
+/* Desktop */
+
+@media only screen and (min-width: 1200px) {
+    header {
+        background-image: url(large_code_skills.jpg);
+    }
+    li {
+        float: right;
+        list-style: none;
```
Comment (charliecog) [inline styles.css]: already declared above - not necessary

**Review by charliecog — CHANGES_REQUESTED** [review]
looking good, just a few changes to make

File: index.html (hunk 1 above)
Comment (JamesCullen2) [reply index.html]: removed <nav>

File: index.html (hunk 2 above)
Comment (JamesCullen2) [reply index.html]: Removed <div>

File: index.html (hunk 4 above)
Comment (JamesCullen2) [reply index.html]: cool, I've taken them out

File: styles.css [hunk 22]
```diff
@@ -131,14 +120,4 @@ section {
     .footer-nav-buttons {
         width: 30%;
     }
- }
```
Comment (charliecog) [inline styles.css]: need to keep that last } in

**Review by charliecog — CHANGES_REQUESTED** [review]
looks good - 1 last change, your almost there!

**Review by charliecog — APPROVED** [review]
looks good - approved, well done james

**Pull request merged**